contracts/tokenableROSCA.sol

```diff
@@ -376,18 +405,27 @@ contract ROSCA {
     uint256 maxAllowedBid = winnerAddress == 0
         ? contributionSize * membersAddresses.length
         : lowestBid * MAX_NEXT_BID_RATIO / 100;
-    if (bidInWei > maxAllowedBid) {
+    if (bid > maxAllowedBid) {
       // We don't throw as this may be hard for the frontend to predict on the
       // one hand because someone else might have bid at the same time,
       // and we'd like to avoid wasting the caller's gas.
-      LogUnsuccessfulBid(msg.sender, bidInWei, lowestBid);
+      LogUnsuccessfulBid(msg.sender, bid, lowestBid);
       return;
     }
-    lowestBid = bidInWei;
+    lowestBid = bid;
     winnerAddress = msg.sender;
     LogNewLowestBid(lowestBid, winnerAddress);
   }
 
+  // Sends funds (either ETH or tokens) to msg.sender. Returns whether successful.
+  function sendFundsToMsgSender(uint256 value) internal returns (bool) {
+    bool isEthRosca = (tokenContract == address(0));
```

Comment:
iirc I think you can use modifiers to instantiate the isEthRosca, just a thought since this check is in a couple of places.

But I do think that using a modifier for something this this reduces readbility, I like the current approach better